src/dataset/domain/models/Dataset.ts

```diff
@@ -16,10 +16,23 @@ export interface License {
   uri: string
   iconUrl?: string
 }
+export interface Citation {
+  authors: string[]
+  creationYear: number
+  title: string
+  persistentIdentifier: string
+  persistentIdentifierUrl: string
+  publisher: string
+  version: string
+  UNF?: string
+  isDeaccessioned?: boolean
+}
```

Comment:
We could simplify the UI model to

```
export interface Citation {
   value: string
   status: Published | Draft | Deaccesioned
   version: string | null
 }
```

```
const citationExample = {
   value: 'Bennet, Elizabeth; Darcy, Fitzwilliam, 2023, "Test Terms", [https://doi.org/10.70122/FK2/KLX4XO](https://doi.org/10.70122/FK2/KLX4XO), Demo Dataverse',
   status: citationStatus.DRAFT
   version: null
}
```

If we make the transformation from the raw data to the UI model when we receive the API data from the js-dataverse module

So we don't have too much logic in the UI component

Reply:
Actually I don't know if the status and version refer to the citation or to the Dataset, if they refer to the Dataset then it would be something like:

interface Dataset {
citation: string
status: Published | Draft | Deaccesioned
version string
}

Reply:
Ok, that makes sense, will update the interface 👍

Reply:
> 
> I left some comments and also I think there is some problem with the Dataset icon because I can only see a blue box
> 
> ![image](https://user-images.githubusercontent.com/23359572/239266164-5318402b-6052-4604-bd80-6888eb20209e.png)
> 
The Dataset Icon was displaying ok until I merged, and now that the Icon is in the design system it is just a blue box. I'm not sure how to fix it,  it seems like it is missing the Icon implementation...can we export that from the Design System?